services/mastodon/mastodon-follow.service.js

```diff
@@ -58,22 +54,27 @@ export default class MastodonFollow extends BaseJsonService {
       message: metric(followers),
       style: 'social',
       link: [
-        `${domain}/users/${username}`,
-        `${domain}/users/${username}/followers`,
+        `https://${domain}/users/${username}`,
+        `https://${domain}/users/${username}/followers`,
       ],
     }
   }
 
   async fetch({ id, domain }) {
     return this._requestJson({
       schema,
-      url: `${domain}/api/v1/accounts/${id}/`,
+      url: `https://${domain}/api/v1/accounts/${id}/`,
     })
   }
 
-  async handle({ id }, { domain = 'https://mastodon.social' }) {
+  async handle({ id }, { domain = 'mastodon.social' }) {
     if (isNaN(id))
       throw new NotFound({ prettyMessage: 'invalid user id format' })
+    if (domain.startsWith('https://')) {
+      domain = domain.substring(8)
+    } else if (domain.startsWith('http://')) {
+      domain = domain.substring(7)
+    }
```

Comment:
Style point: This will work as written, but I'd probably express it as a one-liner:

```suggestion
    domain = domain.replace(/^https?:\/\//, '')
```